app/components/groups/group.contoller.js

```diff
@@ -0,0 +1,102 @@
+;(function(){
+  angular
+    .module("todo")
+
+    .controller("GroupController", GroupController)
+
+    GroupController.$inject = ["$scope"]
+
+    function GroupController($scope){
+      $scope.groupView = "list"
+      $scope.groups = [{
+          title: "First",
+          tasks: [{description: "firstklrgsdgsdgfkdgfkldgfkdgf sadasdsasddas asdasdasd godgfpodgf po asdp[odasokdfigjdfisuj dfsihuodf shjuodhfisjl] description"}],
```

Comment:
Don't leave any data in controller. create a service, where you make calls to server for data and you can put there dummy data also. BTW, you can use $httpBackend for unit tests and mock this call while no server code is ready